### ISSUE TITLE
atty advisory issued

### ISSUE THREAD

**Issue by ckoehler** [bheisler/criterion.rs#629]
See https://rustsec.org/advisories/RUSTSEC-2021-0145

**Comment by briansmith** [comment]
PR ##628 attempts to fix this.

**Comment by briansmith** [comment]
From the duplicate issue #650 (which should be closed):
```
$ cargo audit -Dwarnings
    Fetching advisory database from `https://github.com/RustSec/advisory-db.git`
      Loaded 499 security advisories (from /home/vext01/.cargo/advisory-db)
    Updating crates.io index
    Scanning Cargo.lock for vulnerabilities (137 crate dependencies)
Crate:     atty
Version:   0.2.14
Warning:   unsound
Title:     Potential unaligned read
Date:      2021-07-04
ID:        RUSTSEC-2021-0145
URL:       https://rustsec.org/advisories/RUSTSEC-2021-0145
Dependency tree:
atty 0.2.14
└── criterion 0.4.0

error: 1 denied warning found!
```

According to PR #628, `atty` is no longer maintained. PR #628 attempts to fix this issue by switching to the `is-terminal` crate.

**Comment by waywardmonkeys** [comment]
This can be closed. It was fixed in #628.
